### PR TITLE
create controllers target and event

### PR DESCRIPTION
**Pull request by SetterVet**

### CODE REVIEW

**Comment by SetterVet** [comment]
image

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: %i[update destroy]
+
+  # POST /events
```
Comment (VasylYaremkiv) [inline app/controllers/events_controller.rb]: remove scaffold comments

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  # POST /events.json
+  def create
+    @event = Event.new(event_params)
+
+    respond_to do |format|
+      if @event.save
+        format.json { render template: 'users/show' }
```
Comment (VasylYaremkiv) [inline app/controllers/events_controller.rb]: Why do you render template 'users/show'?

File: app/controllers/events_controller.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  private
+
+  # Use callbacks to share common setup or constraints between actions.
+  def set_event
+    @event = Event.find(params[:id])
+  end
+
+  # Never trust parameters from the scary internet, only allow the white list through.
+  def event_params
+    params.fetch(:event, {})
```
Comment (VasylYaremkiv) [inline app/controllers/events_controller.rb]: Use strong params: https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters

File: app/controllers/targets_controller.rb [hunk 4]
```diff
@@ -0,0 +1,50 @@
+class TargetsController < ApplicationController
```
Comment (VasylYaremkiv) [inline app/controllers/targets_controller.rb]: The same comments here.

**Comment by SetterVet** [comment]
fixed @VasylYaremkiv

**Pull request merged**